syslog-ng/main.c

```diff
@@ -264,9 +264,14 @@ main(int argc, char *argv[])
 
   app_post_daemonized();
   app_post_config_loaded();
-  
+
+  if(startup_debug_flag && !debug_flag)
+  {
+      log_stderr = FALSE;
+  }
+
```

Comment:
hmm.. resetting log_stderr here doesn't really help much, we should reset debug_flag instead (once we remove startup_debug_flag from all code-sites)